primary key can be changed if index is empty (#1597)
meilisearch/documentation: reference/api/indexes.md

> If a primary key wasn't explicitly chosen during index creation, you can use this route to configure it. If you are unsure whether an index's primary key has already been configured, you can use the [get index endpoint](/reference/api/indexes.md#get-one-index) to verify it.

Reviewer: ```suggestion
```

> You can freely update the primary key of an empty index.

Reviewer: ```suggestion
You can freely update the primary key of an index as long as it contains no documents.
```

> You can also change the primary key of an index that already contains documents. To do that, you must first delete all documents in that index. You must then change the primary key, and finally index your dataset again.

Reviewer: ```suggestion
To change the primary key of an index that already contains documents, you must first delete all documents in that index. You may then change the primary key and, finally, index your dataset again.
```

If a primary key wasn't explicitly chosen during index creation, you can use this route to configure it. If you are unsure whether an index's primary key has already been configured, you can use the [get index endpoint](/reference/api/indexes.md#get-one-index) to verify it.

> You can update the primary key of an empty index. If your index contains documents, you will need to delete all of them first, then change the primary key, and add them again.

Reviewer 2: At first I just wanted to separate empty/non-empty cases, but then the editing bug got me. Is this better?

```suggestion
You can freely update the primary key of an empty index.

You can also change the primary key of an index that already contains documents. To do that, you must first delete all documents in that index. You must then change the primary key, and finally index your dataset again.
```
